### PR TITLE
Build full node config path in systemd_units tasks.

### PR DESCRIPTION
**Pull request by abutcher**
The `openshift_node_config_file` var isn't available from `openshift_node` role vars when included in upgrade playbooks with ansible 2.2 rc1.

@sdodson ptal

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

File: roles/openshift_node/tasks/systemd_units.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     - regex: '^OPTIONS='
       line: "OPTIONS=--loglevel={{ openshift.node.debug_level | default(2) }}"
     - regex: '^CONFIG_FILE='
-      line: "CONFIG_FILE={{ openshift_node_config_file }}"
+      line: "CONFIG_FILE={{ openshift.common.config_base }}/node/node-config.yaml }}"
```
Comment (sdodson) [inline roles/openshift_node/tasks/systemd_units.yml]: bad closing braces?

**Comment by abutcher** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Pull request merged**